features/security-monitoring.md

```diff
@@ -0,0 +1,28 @@
+# Security Monitoring
+The Private Packagist security monitoring feature analyzes the dependencies of your private packages for known security vulnerabilities.
+Packages get analyzed on every commit and every time a database reports new vulnerabilities.
+
+The following databases are used to analyze your packages:
+* [sensiolabs/security-advisories](https://github.com/FriendsOfPHP/security-advisories)
+
+### Configure monitoring settings
+The default branch of every private package gets automatically monitored if a composer.lock file is present.
+Additional branches can be configured on the package page.
+
+The feature can be disabled for individual packages on the package security page or for all packages
+on the organization’s security settings page.
+Organizations using the agency add-on also have the option to disable the feature for individual subrepositories.
+
+### Configure notifications
```

Comment:
Headline all caps please, same for the rest here, I think it should also be "Configurating Notifications" rather than "Configure notifications"? same for the other ones.